cmd/cluster/aws/create_test.go

```diff
@@ -81,3 +82,41 @@ func Test_ValidateCreateCredentialInfo(t *testing.T) {
 		})
 	}
 }
+
+func Test_validateMultiArchRelease(t *testing.T) {
```

Comment:
For convention s/Test_validateMultiArchRelease/TestValidateMultiArchRelease. I know the upper one is also with _ but I think the rest of tests all uses camel case.

Reply:
Fixed